CMakeLists.txt

```diff
@@ -5,12 +5,24 @@ set(CMAKE_CXX_STANDARD 14)
 
 option(P2T_BUILD_TESTS "Build tests" OFF)
 option(P2T_BUILD_TESTBED "Build the testbed application" OFF)
+option(BUILD_SHARED_LIBS "Build as a shared library" OFF)
```

Comment:
I think this is built-in and you don't need to add it as an option.

Reply:
You're correct. I've removed the line.